pyproject.toml

```diff
@@ -97,6 +97,8 @@ google-analytics-data = "0.12.1"
 google-api-python-client = "^2.58.0"
 oauth2client = "^4.1.3"
 redis-pal = "^1.0.0"
+datetime = "^5.1"
+unicode = "^2.9"
```

Comment:
`datetime` é um `built-in` do Python, não precisa de importação. Verificarse `unicode` não é o mesmo caso.

Reply:
confesso que não conheço algum comando do `poetry`. verifique e, caso não encontre, apenas elimine as linhas do `pyproject.toml` e, caso tenha gerado, do `poetry-lock.yml`